src/streaming.jl

```diff
@@ -82,7 +82,7 @@ cb_streaming_start_element = cfunction(streaming_start_element, Void, (Ptr{Void}
 function streaming_end_element(p_h::Ptr{Void}, name::Ptr{UInt8})
     h = unsafe_pointer_to_objref(p_h)::XPStreamHandler
     txt::AbstractString = unsafe_string(name)
-    @DBG_PRINT("End element: $txt, current element: $(xph.pdata.name) ")
```

Comment:
again not sure what `xph` refers to since there is no local `xph`  variable in the name